process_report/invoices/billable_invoice.py

```diff
@@ -1,12 +1,12 @@
 from dataclasses import dataclass
-from decimal import Decimal
 import logging
 import sys
 
 import pandas
 import pyarrow
 
 import process_report.invoices.invoice as invoice
+import process_report.invoices.discount_invoice as discount_invoice
```

Comment:
nitpick: it is rare to see `import a.b as b` because a more concise syntax is `from a import b`.

It avoids you having to repeat `b` twice, but it also allows you to specify two modules in one line. 

```
from process_report.invoices import invoice, discount_invoice
```